src/guide/extras/render-function.md

```diff
@@ -644,6 +644,42 @@ const vnode = withDirectives(h('div'), [
 
 Si la directiva está registrada por su nombre y no se puede importar directamente, se puede resolver utilizando el helper [`resolveDirective`](/api/render-function#resolvedirective).
 
+### Referencias de plantilla (template refs)
```

Comment:
no se si dejar el "(template refs)" del título

Reply:
Yo estoy de acuerdo contigo. Creo que es mejor dejarlo. Qué piensas @icarusgk ?

Reply:
Estoy de acuerdo, dejemoslos como (template refs). Solo tendriamos que cambiar "Referencias de plantilla" por "Refs de la Plantilla" para ser congruentes por como iba el resto de la documentación.
<img width="428" alt="Screenshot 2023-12-14 at 16 53 01" src="https://github.com/icarusgk/vuejs-spanish-docs/assets/38413630/0db139a7-1c34-4d0e-b6b6-a0556dc3d552">

Reply:
Me hace ruido ese "de la plantilla", porque es que en realidad no es una plantilla a la que se refiere, sino al bloque "template" de un SFC.

En [escuelavue.es](https://escuelavue.es/tips/template-refs-vue-3/) tampoco lo traducen.

Reply:
Same tbh, suena out of place y que no se refiere a lo que se tiende a referir originalmente en ingles.

En el futuro lo podemos mejorar a una traducción más apta porque por los momentos no suena bien, pero para llevar una traducción congruente dejemoslo asi por ahora, que opinan?

Reply:
En ese caso, es bueno administrar un todo para no olvidar estos detalles.

Reply:
Por supuesto, gracias por bringing it up!